src/handler/global/remind.ts

```diff
@@ -2,39 +2,34 @@ import { SlashCommandBuilder } from "@discordjs/builders";
 import { CommandInteraction } from "discord.js";
 import { UserSettingsRepository } from "../../repository/UserSettingsRepository";
 
-import { Command, CommandData, Guard } from "../../decorators";
+import { CommandHandler, Command, CommandData, Guard } from "../../decorators";
 import { dmGuard } from "../../guard/dmGuard";
 import { Reminder } from "../../entity/Reminder";
 import { AppDataSource } from "../..";
 import { UserSettings } from "../../entity/UserSettings";
 
+@CommandHandler({ name: "remind" })
+@Guard(dmGuard)
```

Comment:
does this need dm guard? pleg was complaining i think (i know it had it before and not really in this scope of this pr but i just remembered lol)